pages/index.html

```diff
@@ -4,9 +4,49 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Mexican Cousine</title>
+    <link href="styles/style.css" rel="stylesheet">
+    <link href="styles/style-index.css" rel="stylesheet">
 </head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+<body class="container">
+  <header>
+    <nav class="nav-bar">
+      <a href="facts.html">Fun Facts!</a>
+      <a href="gallery.html">Gallery!</a>
+    </nav>
+  </header>
+    <h1 class="shimmer">Mexican Cuisine: A Heritage of Tradition</h1>
+    <img id="header" src="images/mexican-foods-tamales.webp" alt="Mexican Tamales" />
+    <p class="main-text">Traditional Mexican cuisine is so much more than authentic dishes and spices; it's a vibrant, lovingly preserved, cultural heritage.</p>
+    <br>
+    <p>Planting and harvesting native ingredients such as corn and chile peppers in chinampas…
+      Manually grinding grain for tortillas on stone and mortar…
+      <br>
+      <br>
+      <br>
+      Traditional Mexican cuisine embraces an entire heritage of culture dedicated to maintaining Mexico’s indigenous customs and cultural traditions.</p>
+    <img class="second" src="images/mexican-food.jpeg" alt="Mexican food">
+    <div class="final">
+      <p>Mexican cuisine it's now internationally recognized as an Intangible Cultural Heritage of Traditional Cuisine!</p>
+        <br>
+        <br>
+        <p id="text">This content was created by:</p>
```

Comment:
It's a good idea to indent things that are siblings at the same level. The way this is currently indented, I thought the `br` and `p` tags were nested inside of the top `p` tag at first.

```suggestion
      <p>Mexican cuisine it's now internationally recognized as an Intangible Cultural Heritage of Traditional Cuisine!</p>
      <br>
      <br>
      <p id="text">This content was created by:</p>
```